java-client/src/main/java/org/apache/pegasus/client/PegasusTable.java

```diff
@@ -98,6 +128,10 @@ public Future<Long> asyncSortKeyCount(byte[] hashKey, int timeout) {
         new Table.ClientOPCallback() {
           @Override
           public void onCompletion(client_operator clientOP) {
+            if (!(clientOP instanceof rrdb_sortkey_count_operator)) {
+              promise.setFailure(new PException("Unexpected client operator"));
```

Comment:
Better to not expose netty's promise and future. We can use CompletableFuture in JDK.  Can be a separated issue, and this will be an incompatible change...

Reply:
Consistent with the above c-format problem, the current usage is like this, I think the follow-up pr can be optimized.